cumulus_library/cli.py

```diff
@@ -392,6 +390,36 @@ def main(cli_args=None):
                 args[pair[0]] = env_val
             read_env_vars.append([pair[1], env_val])
 
+    if args.get("study_dir"):
+        posix_paths = []
+        for path in args["study_dir"]:
+            posix_paths.append(get_abs_path(path))
+        args["study_dir"] = posix_paths
+
+    if args.get("action") == "study-version":
+        studies = get_study_dict(args["study_dir"])
+        for target in args.get("target"):
+            if target in ("core", "discovery", "vocab"):
+                print(
+                    f"{target} distributed with Cumulus library.\n"
+                    f"Cumulus library:  {__version__}"
+                )
```

Comment:
I'm a little leery of this special logic here. The user can point at a different / in-development version of these built-in studies with `-s`, where this may not give the right version. And `vocab` is slated for movement - so depending on how that goes down, old versions of `library` may not give the correct version after the move, especially if `-s` is involved again.

I understand you're doing that so that you don't have to keep track of versions inside core/discovery/vocab. But also... maybe we should? Like, `discovery` has its own semantic version lifecycle separate from `core`, which in theory lives a different versioning life from the outer library study runner (though in practice, those two are often closely entwined).

Anyway, just a nit comment, this is fine for now.

Reply:
Well I wasn't even talking about repos - just a version in their `__init__.py` files

Reply:
But the separate-version approach also matters less if we do a "all versions printed" approach from below.

Reply:
this was sort of a side thought - i'm doing the all versions thing below for now, i guess i'm going to pin core to a v3 since it's mostly tied to the core infra and discovery to 1.0, but i could see a decoupling case generally? not taking action on it atm.